src/Docfx.App/Helpers/PlaywrightHelper.cs

```diff
@@ -0,0 +1,43 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Runtime.InteropServices;
+using Docfx.Common;
+using Docfx.Exceptions;
+
+namespace Docfx;
+
+internal static class PlaywrightHelper
+{
+    public static void EnsurePlaywrightNodeJsPath()
+    {
+        // Skip if playwright environment variable exists.
+        if (Environment.GetEnvironmentVariable("PLAYWRIGHT_DRIVER_SEARCH_PATH") != null)
+            return;
+
+        if (Environment.GetEnvironmentVariable("PLAYWRIGHT_NODEJS_PATH") != null)
+            return;
+
+        // Find Node.js executable installation path from PATHs.
+        string exeName = RuntimeInformation.IsOSPlatform(OSPlatform.Windows) ? "node.exe" : "node";
```

Comment:
Does this work here ? https://github.com/dotnet/docfx/blob/main/src/Docfx.Common/CommandUtility.cs#L110C24-L110C36

Reply:
On latest commit.
Refactor code to use `CommandUtility`.

Additionally. I've added log for Node.js version.